CarpetLib/src/restrict_3d_cc_o3_rf2.cc

```diff
@@ -215,8 +215,8 @@ void restrict_3d_cc_o3_rf2(
 
   } else {
 
-// Loop over coarse region
-#pragma omp parallel
+    // Loop over coarse region
+    // #pragma omp parallel
```

Comment:
this should not be committed in this form. Rather the whole pragma should be removed.